linphone.xcodeproj/project.pbxproj

```diff
@@ -2487,11 +2491,14 @@
 				220FAD3910765B400068D98F /* libspeexdsp.a in Frameworks */,
 				226183AE1472527D0037138E /* libsrtp.a in Frameworks */,
 				22276E8513C73D3100210156 /* libswscale.a in Frameworks */,
+				63D7215D1B7394D200D70E65 /* libtunnel.a in Frameworks */,
```

Comment:
belledonne added reading from a xcode file and extracting it from there.. that where all the trouble began. I missed one line  when merging the new stuff  https://github.com/shareefalis/ace-iphone/commit/fa222a3ca1cd9e8372fef9b87e52e34191495ff6